lib/cdo/hamburger.rb

```diff
@@ -107,16 +103,10 @@ def self.get_hamburger_contents(options)
       entry[:title] = I18n.t("#{loc_prefix}#{entry[:title]}")
     end
 
-    signed_out_entries = [
```

Comment:
This is removing these:

<img width="300" alt="Screenshot 2024-06-05 at 12 59 39 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/2730229c-2719-4247-85c4-b5f6a65faaef">